.gitignore

```diff
@@ -3,3 +3,4 @@ resources/.DS_Store
 .DS_Store
 resources/magic-egg-allinone-exporter.mse-export-template/.DS_Store
 resources/magic-egg-allinone-exporter.mse-export-template/.DS_Store
+**/__pycache__
```

Comment:
just `__pycache__` should pick up all instances, the glob before it is not needed